src/app/validators/recaptchaValidation.ts

```diff
@@ -0,0 +1,44 @@
+import {ValidatorConstraint, ValidatorConstraintInterface, ValidationArguments} from "class-validator";
+
+@ValidatorConstraint({ name: "IsCaptcha", async: true })
+export class IsCaptcha implements ValidatorConstraintInterface {
+
+	private async checkRecaptcha(value) {
+        if (!value) {
+            return new Promise((resolve, reject) => {
+                resolve({});
+            });
+        } else {
+            const Recaptcha = require('recaptcha-verify');
```

Comment:
rather than require this should probably be an import statement at the top I think import * as  Recaptcha from 'recaptcha-verify'; should work.